docs/networking/connecting-the-arena-to-a-network.md

```diff
@@ -1,37 +1,43 @@
 ---
 id: connecting-the-arena-to-a-network 
-title: Connecting the Arena to a network
-sidebar_label: Connecting the Arena to a network
+title: Connecting the Diamond 9 or Arena to a network
+sidebar_label: Connecting the Diamond 9 or Arena to a network
 ---
 
-The Arena console is different to the other Titan consoles because it
-has a built-in network switch and an optical network connection
-(optionally 2 optical connections). **This section applies only to the
-Arena console.**
+The Diamond 9 and Arena consoles are different to the other Titan consoles because they
+have a built-in network switch and an optical network connection. 
+**This section applies only to the Diamond 9 and 
+Arena consoles.**
 
 Optical Connections
 -------------------
 
-The optical network connector is a Neutrik opticalCON Duo which uses
-multimode fibre. A single connector is fitted as standard, but a second
-can be fitted on request. The stage end of the connection normally plugs
+The optical network connector is a Neutrik opticalCON which uses
+multimode fibre. On the Arena the connector is a Duo, a single connector is fitted as standard, but a second
+can be fitted on request. On the Diamond 9 the connector is a Quad with one of the two pairs populated as standard - if the
+optional 10GbE Ethernet is fitted then the second pair is populated with a 10GbE fibre.
+
+The stage end of the connection normally plugs
 into the Avolites TitanNet Switch (TNS) rack unit which provides
 standard wired Ethernet connections. If you want to connect to a
 different optical switch you should contact Avolites for advice.
 
 Wired Ethernet Outputs
 ----------------------
 
-The Arena console has an in-built 1 Gb TitanNet switch (TNS) that is
+The Diamond 9 console has an in-built 1Gb Luminex network switch; the 
+Arena console has an in-built 1 Gb TitanNet switch (TNS). These are
```

Comment:
Sometimes 1 Gb other times 1Gb. Would be good to be consistent but not the end of the world